dicoogle/src/main/java/pt/ua/dicoogle/server/web/servlets/management/ForceIndexing.java

```diff
@@ -60,7 +60,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
 
     // Getting Parameters.
     String[] uris = req.getParameterValues("uri");
-    String[] pluginsName = req.getParameterValues("plugin");
+    String[] pluginsName = req.getParameterValues("plugin[]");
```

Comment:
We shouldn't be doing this. Modifying the server's API is a breaking change (and in this case, an unnecessary one). We should instead adjust the web client to build multiple "plugin" query string parameters (e.g. `/foo?plugin=x&plugin=y`).